### PR TITLE
feat: update invite clipboard text

### PR DESCRIPTION
**Pull request by domw30**
### What does this do?
- updates the clipboard text when copying invite code

### Why are we making this change?
- as per requirements/design
- this gives more context/info to the person who receives the invite code that a current user has copied and sent.

### How do I test this?
- run test as usual.
- checkout branch > run UI > copy invite code > paste and check string has the same structure as below screenshot.

#### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Screenshot 2024-03-13 at 17 58 14

### CODE REVIEW

File: src/components/invite-dialog/index.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export class InviteDialog extends React.Component<Properties, State> {
   }
 
   get inviteText() {
-    return this.props.inviteCode;
+    return `Use this code to join me on ZERO Messenger: ${this.props.inviteCode} https://zos.zer0.io/get-access`;
```
Comment (dalefukami) [inline src/components/invite-dialog/index.tsx:48]: We definitely shouldn't hard code the url. Do we not have env vars for the root url?

Comment (domw30) [reply src/components/invite-dialog/index.tsx:48]: We'll have to add a new one for the root url

Comment (dalefukami) [reply src/components/invite-dialog/index.tsx]: Seems like we have REACT_APP_MESSENGER_INVITE_PATH

Comment (domw30) [reply src/components/invite-dialog/index.tsx]: Sure, I'll update.

Comment (domw30) [reply src/components/invite-dialog/index.tsx]: Updated to use REACT_APP_MESSENGER_INVITE_PATH & removed the env var I added to github variables. Changes merged :)

**Pull request merged**